packages/venia-concept/src/components/CreateAccountPage/helpers.js

```diff
@@ -0,0 +1,4 @@
+import qs from 'qs';
+
+export const getCreateAccountInitialValues = () =>
+    qs.parse(window.location.search, { ignoreQueryPrefix: true });
```

Comment:
Please replace this usage of `qs` with [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) as well.